sw/device/silicon_creator/lib/sigverify_mod_exp_ibex.c

```diff
@@ -166,6 +166,12 @@ static void calc_r_square(const sigverify_rsa_key_t *key,
 rom_error_t sigverify_mod_exp_ibex(const sigverify_rsa_key_t *key,
                                    const sigverify_rsa_buffer_t *sig,
                                    sigverify_rsa_buffer_t *result) {
+  // Reject the signature if it is too large (n <= sig): RFC 8017, section
+  // 5.2.2, step 1.
+  if (memrcmp(key->n.data, sig->data, kSigVerifyRsaNumBytes) <= 0) {
```

Comment:
I think we can use `greater_equal_modulus()` here but not in `sigverify_mod_exp_otbn.c`.

Reply:
True! I'll change this one.